nixos/modules/services/misc/autorandr.nix

```diff
@@ -28,7 +38,15 @@ in {
       wantedBy = [ "sleep.target" ];
     };
 
+    nixpkgs.overlays = mkIf (cfg.defaultTarget != null) [
```

Comment:
Why an overlay and not a local package override?

Reply:
you're actually right, I thought that it might sense to override the package when the option is set in the module system, but it's actually not needed as it only alters the `.service` file :)